pytensor/tensor/nlinalg.py

```diff
@@ -1026,3 +1027,45 @@ def tensorsolve(a, b, axes=None):
     "tensorinv",
     "tensorsolve",
 ]
+
+
+# Adding the kron function here, which earlier used to be in slinalg.py
+def kron(a, b):
+    """Kronecker product.
+
+    Same as np.kron(a, b)
+
+    Parameters
+    ----------
+    a: array_like
+    b: array_like
+
+    Returns
+    -------
+    array_like with a.ndim + b.ndim - 2 dimensions
+
+    Notes
```

Comment:
Sorry I wasn't clear before, I meant delete this whole Note. It's not relevant to users that scipy kron is broken